README.md

```diff
@@ -26,7 +26,7 @@ The release is called **v1.8.1**, an update to add preliminary support to Open B
 
 ---
 **Download**:
-`docker run --rm -it -p 127.0.0.1:8443:8443 "openbanking/conformance-suite:v1.8.0"` |
+`docker run --rm -it -p 127.0.0.1:8443:8443 "openbanking/conformance-suite:v1.8.1"` |
```

Comment:
leaving in the main README the latest stable version (works with v3.0) now iterating with draft releases "beta" until we fix all errors

Reply:
Sounds good to me